project/LagomPublish.scala

```diff
@@ -15,18 +16,22 @@ object LagomPublish {
       case ("lagom-sbt-plugin", x) =>
         // see https://github.com/sbt/sbt-bintray/blob/7c93bacaae3ffc128564ceacb6e73ec4486525dd/src/main/scala/Bintray.scala#L16-L29 for
         // details on the syntax of Bintray Resolver names.
-        if (x.get.name != "Bintray-Sbt-Publish-lagom-sbt-plugin-releases-lagom-sbt-plugin") {
-          throw new PublishValidationFailed("""Invalid resolver. Expected: "Raw(Bintray-Sbt-Publish-lagom-sbt-plugin-releases-lagom-sbt-plugin)".""")
+        if (inReleaseVersion && x.get.name != "Bintray-Sbt-Publish-lagom-sbt-plugin-releases-lagom-sbt-plugin") {
+          throw new PublishValidationFailed(s"""Invalid resolver. Expected: "Raw(Bintray-Sbt-Publish-lagom-sbt-plugin-releases-lagom-sbt-plugin)" but was "${x.get.name}".""")
```

Comment:
The validation for `lagom-sbt-plugin` only passed in released versions because the current settings configure the `Resolver` to something different than `Bintray` when in a `SNAPSHOT`.

We don't support publishing snapshots ATM so I just disabled the validation when not on a release.